### PR TITLE
upgrade to knative build 0.2.0

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

File: knative-build/templates/build-clusterrole.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ kind: ClusterRole
 metadata:
   name: {{ template "build.name" . }}-admin
 rules:
+rules:
```
Comment (garethjevans) [inline knative-build/templates/build-clusterrole.yaml]: this duplicate rules block looks wrong

Comment (rawlingsj) [reply knative-build/templates/build-clusterrole.yaml]: nice thanks

**Pull request merged**